### DISCUSSION TITLE
Ikamand v2.0

### DISCUSSION THREAD

**Discussion by raymondo72** [plmilord/Hass.io-custom-component-ikamand#9] [General]
Firstly @plmilord  - amazing work on this ! I was fuming over KJ dumping this item and deleting the app! 
I'm new to Hacs etc, but Ive got it installed now, and can see probe data on all 3 probes and the connection seems stable
However, the fan will not fire up when controlling from the app
If i press the physical button on the device - the fan does fire up and the app reports the fan speed correctly as 100%
What am I doing wrong :_) ? Im guessing when you press FIRE IT UP it should kick in for however long you set fan duration to 
Thanks!

**Comment by raymondo72** [comment]
Ignore - I didnt have a pit probe connected as I was testing ! plugged in and it works great . Amazing !

**Comment by plmilord** [comment]
No worries! Happy BBQ time!
